src/animations/Animation.js

```diff
@@ -620,10 +641,24 @@ var Animation = new Class({
         {
             //  We're at the start of the animation
 
-            if (component.repeatCounter > 0)
+            if (component._yoyo)
             {
-                //  Repeat (happens before complete)
-                this.repeatAnimation(component);
+                component.forward = true;
+                this._updateAndGetNextTick(component, frame.nextFrame);
+            }
+            else if (component.repeatCounter > 0)
+            {
+                if(component._reverse && !component.forward)
```

Comment:
Please quickly run this against ESLint (`npm run lintfix`) as it'll break the keywords-spacing rule.

Reply:
Idk if it is a problem with my eslint but it's not showing errors of keywords-spacing...